WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -230,6 +230,13 @@ private extension BlogDashboardViewModel {
         return snapshot
     }
 
+    var isShowingQuickActions: Bool {
+        guard Feature.enabled(.sidebar) else {
+            return !WPDeviceIdentification.isiPad()
+        }
+        return viewController?.traitCollection.horizontalSizeClass == .compact
```

Comment:
What do you think checking the `dataSource` instead, which should be the true representation of the view? i.e. `dataSource.snapshot().sectionIdentifiers.contains(.quickActions)`.

My concern with the current check is that we may change when to display quick actions in the future, but forget to update this function here.

Reply:
It was a good opportunity to simplify it. Turns out, it was all unnecessary, and I removed this code in https://github.com/wordpress-mobile/WordPress-iOS/pull/23593/commits/9514c90e1c7c3c390333360d84d1bd668df51e98, removed one of `isPad` usage, and added a bit of an bottom inset on `.regular` screen size.

<img width="800" alt="Screenshot 2024-09-13 at 9 21 51 AM" src="https://github.com/user-attachments/assets/9531c825-f92e-41a1-9f0d-b9467eb7aca9">

<img width="800" alt="Screenshot 2024-09-13 at 9 23 05 AM" src="https://github.com/user-attachments/assets/9d44eb96-8e24-49db-a495-6480c5e930f8">